<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>470</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>labelPic</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPic">
    <property name="title">
     <string>目录树</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <addaction name="menuPic"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFolder"/>
   <addaction name="actionAddFiles"/>
   <addaction name="actionDelete"/>
   <addaction name="actionScan"/>
   <addaction name="separator"/>
   <addaction name="actionZoomPlus"/>
   <addaction name="actionZoomMinus"/>
   <addaction name="actionRealSize"/>
   <addaction name="actionFitWidth"/>
   <addaction name="actionFitHeight"/>
   <addaction name="separator"/>
   <addaction name="actionFloat"/>
   <addaction name="actionVisible"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeFiles">
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片文件</string>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
         <item>
          <property name="text">
           <string>图片节点</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组2</string>
         </property>
         <item>
          <property name="text">
           <string>图片2</string>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/new-folder.png</normaloff>:/img/new-folder.png</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/add-file.png</normaloff>:/img/add-file.png</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionZoomPlus">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/zoom_plus.png</normaloff>:/img/zoom_plus.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionZoomMinus">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/zoom_minus.png</normaloff>:/img/zoom_minus.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionRealSize">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/smart-tv.png</normaloff>:/img/smart-tv.png</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionFitWidth">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/width.png</normaloff>:/img/width.png</iconset>
   </property>
   <property name="text">
    <string>适应宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionFitHeight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/height.png</normaloff>:/img/height.png</iconset>
   </property>
   <property name="text">
    <string>适应高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/scanner.png</normaloff>:/img/scanner.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/floating-island.png</normaloff>:/img/floating-island.png</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/idea.png</normaloff>:/img/idea.png</iconset>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
